features/node/kata/install_uninstall.feature

```diff
@@ -0,0 +1,54 @@
+Feature: kata related features
+  # @author pruan@redhat.com
+  # @case_id OCP-36508
+  @admin
+  @destructive
+  @upgrade-prepare
+  Scenario: kata container operator installation
```

Comment:
If we tag it upgrade-prepare, then we need to postfix it with "- prepare"
Also, we need a scenario for upgrade check.

Reply:
@liangxia took those tags out.

Reply:
@liangxia actually I do need that `tag` even though I'm not doing an upgrade....it's due to I'm callling this [step](https://github.com/openshift/verification-tests/blob/master/features/step_definitions/scc.rb#L65)

Reply:
@akostadinov @liangxia why do we need to postfix it with `-prepare`?  just a file naming convention?